docs/guides/chip_tool_guide.md

```diff
@@ -1393,3 +1393,244 @@ $ ./chip-tool any read-by-id <cluster-ids> <attribute-ids> <destination-id> <end
+    For example, the PIN code value (`12345` in the example below).
+-   _<user-index\>_ is the index of the user that will be associated with the
+    credential. Can be set to `null`.
+-   _<user-status\>_ is the status of the user that will be associated with the
+    credential. See the description of this parameter in
+    [Set up a user](#step-1-set-up-a-user). Can be set to `null`.
+-   _<destination-id\>_ is the ID of the door lock device.
+-   _<endpoint-id\>_ is the ID of the endpoint on the door lock device.
+-   `--timedInteractionTimeoutMs` is the timeout duration for the interaction,
+    in milliseconds (_<ms_value>_). This should allow enough time for the
+    interaction to complete.
+
+**Example of command:**
+
+```
+$ ./chip-tool doorlock set-credential 0 '{ "credentialType": 1, "credentialIndex": 1 }' "12345" 1 null null 1 1 --timedInteractionTimeoutMs 1000
+```
+
+### Operations on users and credentials
+
+After you set up users and credentials on your door lock device, you can use
+several CHIP Tool commands to interact with them.
+
+All commands reuse the parameters explained earlier in this section. The
+following command patterns are available:
+
+-   Reading the status of the user:
+
+    ```
+    $ ./chip-tool doorlock get-user <user-index> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
+    ```
+
+    This command returns the status of the user at the specified _<user-index\>_
+    at the specified _<destination-id\>_ and _<endpoint-id\>_.
+
+-   Reading the status of the credential:
+
+    ```
+    $ ./chip-tool doorlock get-credential-status <{Credential}> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
+    ```
+
+    This command returns the status of the credential of the specified
+    _<{Credential}\>_ at the specified _<destination-id\>_ and _<endpoint-id\>_.
+
+-   Cleaning the user:
+
+    ```
+    $ ./chip-tool doorlock clear-user <user-index> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
+    ```
+
+    This command cleans the slot containing the specified _<user-index\>_ at the
+    specified _<destination-id\>_ and _<endpoint-id\>_.
+
+-   Cleaning the credential:
+
+    ```
+    $ ./chip-tool doorlock clear-credential <{Credential}> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
+    ```
+
+    This command cleans the slot containing the specified _<{Credential}\>_ at
+    the specified _<destination-id\>_ and _<endpoint-id\>_.
+
+### Operations with user PIN code
+
+If you set the _<credential-type\>_ to PIN when
+[assigning credentials](#step-2-assign-a-credential), you can use the following
+command patterns to verify if it works and invoke it to open or close the door
+lock:
+
+-   Verifying the PIN code:
+
+    ```
+    $ ./chip-tool doorlock read require-pinfor-remote-operation <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
```

Comment:
```suggestion
    $ ./chip-tool doorlock read require-pinfor-remote-operation <destination-id> <endpoint-id>
```

you seem to be pretty confused about this `--timedInteractionTimeoutMs` parameter...